docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -0,0 +1,84 @@
+const sass = require('node-sass');
+const glob = require('glob');
+
+module.exports = function (grunt) {
+
+  function getIncludeFiles() {
+    const patterns = [
+      '**/*.scss',
+      'node_modules/decanter/scss',
+    ];
+
+    const libraries = [];
+    patterns.map(function (pattern) {
+      glob.sync(pattern).map(function (file) {
+        libraries.push(file);
+      });
+    });
+
+    return libraries;
+  }
+
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    watch: {
+      css: {
+        files: ['**/*.{scss,sass}'],
+        tasks: ['sass', 'run:stylelint'], // run stylelint at same time after each change
+        options: {
+          interrupt: true
+        }
+      }
+    },
+    run: {
+      stylelint: {
+        exec: 'npm run lint:css'
+      }
+    },
+    postcss: {
+      options: {
+        map: true, // inline sourcemaps
+        processors: [
+          require('autoprefixer')({
+            grid: true,
+          }) // add vendor prefixes
+        ]
+      },
+      dist: {
+        src: [
+          'dist/**/*.css'
+        ]
+      }
+    },
+    sass: {
+      options: {
+        implementation: sass,
+        sourceMap: false,
+        outputStyle: 'compressed',
+        includePaths: getIncludeFiles()
+      },
+      dist: {
+        files: [
+          {
+            expand: true,
+            cwd: 'src',
+            src: ['scss/**/[a-z]*.scss'],
```

Comment:
ohh so placing this under the src folder negates it from the root gruntfile command right?

Reply:
If you are talking about the `Gruntfile` in the root of the repository, then no, this would not negate that at all. We would need to change the root Gruntfile somehow to skip over this theme (and the sub-themes too probably).

Reply:
Nathan's PR for decanter does this.